packages/keyring-eth-ledger-bridge/src/ledger-keyring.ts

```diff
@@ -413,7 +414,26 @@ export class LedgerKeyring implements Keyring {
         tx: remove0x(rawTxHex),
         hdPath,
       });
-    } catch (error) {
+    } catch (error: unknown) {
+      // check whether error is TransportError
+      if (error instanceof TransportStatusError) {
+        // cast error to TransportError
+        const transportError: TransportStatusError = error;
+        if (
+          transportError.statusCode === 27013 &&
+          transportError.message.includes('(denied by the user?) (0x6985)')
+        ) {
+          throw new Error('Ledger: User rejected the transaction');
+        }
+
+        if (
+          transportError.statusCode === 27264 &&
+          transportError.message.includes('Invalid data received (0x6a80)')
+        ) {
+          throw new Error('Ledger: Blind signing must be enabled');
+        }
+      }
```

Comment:
Let's make a function for this, something like `reThrowIfTransportError(error)` maybe?

So we can use it here and in the other `catch` clauses.

Reply:
Will be addressed in a follow-up PR.